scripts/lis_hyperv_platform/main.ps1

```diff
@@ -270,11 +275,17 @@ function Main {
             throw "Creating the LISA VM failed."
         }
 
-        $idRSAPriv = Get-Lisa
+        Write-Host "Getting LISA code..."
+        Get-LisaCode
+        Invoke-WebRequest -Uri $PuttyBinaries -OutFile "PuttyBinaries.zip"
+        if ($LastExitCode) {
+            throw "Failed to download Putty binaries."
+        }
+        Expand-Archive ".\PuttyBinaries.zip" ".\lis-test\WS2012R2\lisa\bin"
```

Comment:
Example: Add a global variable $LISA_REL_PATH=".\lis-test\WS2012R2\lisa\" and use it every time we need something. this will enable us to also change it on the fly if needed.